data/mods/gen7/moves.ts

```diff
@@ -299,20 +299,16 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 					this.add('-fieldstart', 'move: Grassy Terrain');
 				}
 			},
-			onFieldResidualOrder: 5,
+			onFieldResidualOrder: 21,
 			onFieldResidualSubOrder: 3,
```

Comment:
Let's put these next to `onFieldEnd` for conditions without `onFieldResidual`?

Let's also put `onFieldResidual` next to `onFieldEnd` (so they're always the last two) for conditions that do.